values/tekton-pipelines/tekton-pipelines.gotmpl

```diff
@@ -3,28 +3,30 @@
 
 controller:
   resources: {{- $t.resources.pipelinesController | toYaml | nindent 4 }}
-  {{- if $v.otomi.linodeLkeImageRepository }}
   deployment:
-    image: "{{- $v.otomi.linodeLkeImageRepository }}/gcr/tekton-releases/github.com/tektoncd/pipeline/cmd/controller:v0.53.0@sha256:9cef507c33127c488938fd6af10c0c2242b4b667732e488545338f290025fa08"
-  {{- end }}
+    image: "{{- if $v.otomi.linodeLkeImageRepository }}{{- $v.otomi.linodeLkeImageRepository }}/ghcr{{- else }}ghcr.io{{- end }}/tektoncd/pipeline/controller-10a3e32792f33651396d02b6855a6e36:v1.3.1@sha256:702fca76e77ef1dc991d72b41fe7af4be00f0e0c84160060a7bdf11cd6a3429f"
```

Comment:
A bit simpler would be:

```suggestion
    image: "{{ with $v.otomi.linodeLkeImageRepository }}{{ . }}/ghcr{{ else }}ghcr.io{{ end }}/tektoncd/pipeline/controller-10a3e32792f33651396d02b6855a6e36:v1.3.1@sha256:702fca76e77ef1dc991d72b41fe7af4be00f0e0c84160060a7bdf11cd6a3429f"
```

The whitespace control has no effect if it is all on the same line.